__tests__/server/utils/pollModuleMap.spec.js

```diff
@@ -231,6 +222,10 @@ describe('pollModuleMap', () => {
     expect(setTimeout).toHaveBeenCalledTimes(2);
     expect(setTimeout.mock.calls[1][0]).toBe(pollModuleMap);
 
+    setGauge.mockImplementationOnce(() => {
+      throw new Error('unable to increment gague');
```

Comment:
#nitpick
```suggestion
      throw new Error('unable to change gauge');
```
gauges go up and down, _counters_ are incremented 😅